.github/workflows/conformance-e2e.yaml

```diff
@@ -417,7 +417,7 @@ jobs:
         with:
           skip-build: ${{ env.CILIUM_CLI_SKIP_BUILD }}
           image-repo: ${{ env.CILIUM_CLI_IMAGE_REPO }}
-          image-tag: ${{ env.CILIUM_CLI_VERSION }}
+          image-tag: 577aad3380b6a3dd953a7afde9cd2a8ec8383f58
```

Comment:
Is this temporary change?

Reply:
yes, see the commit (https://github.com/cilium/cilium/pull/33434/commits/0bef665c968bec7b5324fc27c78cb3065b8eada1) title for that change